lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -121,6 +121,29 @@ static unsigned getPrintedIntWidth(unsigned value) {
   return stream.str().size();
 }
 
+/// Calculated the printed width of the array dims of a type.
+static size_t getPrintedTypeDimsWidth(Type type, Operation *op) {
+  int bitWidth;
+  size_t textWidth = 0;
+  if (auto array = type.dyn_cast<rtl::ArrayType>()) {
+    bitWidth = array.getSize();
+    textWidth = getPrintedTypeDimsWidth(array.getElementType(), op);
+  } else if (auto inout = type.dyn_cast<rtl::InOutType>()) {
```

Comment:
I'd recommend moving the inout case first and ahead of the declaration of bitwidth since it isn't part of the other's flow, and similarly on the printing method below.